DataFormats/SiPixelDetId/src/classes_def.xml

```diff
@@ -7,4 +7,17 @@
    <version ClassVersion="11" checksum="1218239916"/>
    <version ClassVersion="10" checksum="64579558"/>
   </class>
+  <class name="PixelFEDChannel" ClassVersion="3">
+   <version ClassVersion="3" checksum="2591256255"/>
+  </class>
+  <class name="std::vector<PixelFEDChannel>"/>
+  <class name="edm::EDCollection<PixelFEDChannel>"/>
+  <class name="edmNew::DetSet<PixelFEDChannel>"/>
+  <class name="std::vector<edmNew::DetSet<PixelFEDChannel> >"/>
+  <class name="edmNew::DetSetVector<PixelFEDChannel>"/>
+  <class name="edm::Wrapper<std::vector<PixelFEDChannel> >"/>
+  <class name="edm::Wrapper<edm::EDCollection<PixelFEDChannel> >"/>
+  <class name="edm::Wrapper<edmNew::DetSet<PixelFEDChannel> >"/>
+  <class name="edm::Wrapper<std::vector<edmNew::DetSet<PixelFEDChannel> > >"/>
+  <class name="edm::Wrapper<edmNew::DetSetVector<PixelFEDChannel> >"/>
```

Comment:
are all of these dictionaries really necessary, considering that only ```edmNew::DetSetVector``` is persisted (and only ```std::vector<PixelFEDChannel>``` is apparently also required )?
- is EDCollection needed?
- is ```vector<edmNew::DetSet ``` needed?
- is ```edmNew::DetSet ``` needed?

Reply:
DetSetVector is the one actually used for the bad channel propagation. So I believe EDCollection is the only line above that is not being used (explicitly or implicitly) for the time being; however, if there is no argument against keeping it, it would be a good to do so in case some user does not want to put the overhead of the DetId mapping in the event that otherwise comes with PixelFEDChannelCollection. (PixelFEDChannel is a generic class, not necessarily for what we use it here.)